# Generated by Django 2.2.6 on 2019-10-10 11:47

from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion


class Migration(migrations.Migration):

    initial = True

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
    ]

    operations = [
        migrations.CreateModel(
            name='CronJobResponses',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('cronjob', models.IntegerField()),
                ('cronjob_title', models.CharField(max_length=255)),
                ('url', models.CharField(max_length=255)),
                ('response', models.CharField(max_length=255)),
            ],
        ),
        migrations.CreateModel(
            name='CronJob',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('title', models.CharField(default='', max_length=255)),
                ('url', models.URLField(default='', max_length=255)),
                ('auth_enabled', models.BooleanField(default=False)),
                ('auth_user', models.CharField(max_length=255, null=True)),
                ('auth_pw', models.CharField(max_length=255, null=True)),
                ('execute_interval', models.CharField(default='* * * * *', max_length=255)),
                ('allert_failed', models.BooleanField(default=False)),
                ('allert_success_after_failed', models.BooleanField(default=False)),
                ('allert_too_much_fails', models.BooleanField(default=False)),
                ('save_response', models.BooleanField(default=True)),
                ('creater', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to=settings.AUTH_USER_MODEL)),
            ],
        ),
    ]
